### DISCUSSION TITLE
Adding equations to an existing model in PYBAMM

### DISCUSSION THREAD

**Discussion by sreehari33** [pybamm-team/PyBaMM#2504] [Q&A]
I want to copy an existing model and add new physics to it. could someone please share some resources which would help me?
Thank you

**Comment by RuiheLi** [comment]
Hi~ I think [these notebooks](https://github.com/pybamm-team/PyBaMM/tree/develop/examples/notebooks/Creating%20Models) can precisely help you. Just read through them carefully then you should be able to crack on your equations.
